### PR TITLE
Gatsby GitHub GraphQL API Integration

### PR DESCRIPTION
**Pull request by accbjt**
Adding feature to be able to access and display GitHub Markdown pages directly from github.

### CODE REVIEW

**Comment by accbjt** [comment]
@jbalboni we need a GitHub API token for this to work. What is the process of getting this. We also need it to registerd on Jenkin's for the build to work. I created a ticket for getting the API token.

https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/16880

File: packages/documentation/gatsby-node.js [hunk 1]
```diff
@@ -7,13 +7,44 @@
  */
 
 const path = require('path')
+const githubApi = require('./services/github-api')
```
Comment (accbjt) [inline packages/documentation/gatsby-node.js]: This is a service I created to incapsulate all the request to the Github API or any functions we create to access the GitHub API. I wanted to keep everything contained and have one source of truth.

File: packages/documentation/gatsby-node.js [hunk 2]
```diff
@@ -7,13 +7,44 @@
...
+  getNode,
+  hasNodeChanged,
+}) => {
+  const { createNode } = actions
+  const repoData = {
+    owner: 'department-of-veterans-affairs',
+    repo: 'vets.gov-team',
+  }
+
+  await githubApi.getDirectoryAndCreatePages({
```
Comment (jbalboni) [inline packages/documentation/gatsby-node.js]: We talked about this a bit, I think this is fine for now, but in the future I think we'll want to have a separate file that has a list of pages (and their repo) that we want to pull in.

File: packages/documentation/src/pages/add-external-pages.mdx [hunk 3]
```diff
@@ -0,0 +1,287 @@
+---
+title: How to Add Pages from GitHub
```
Comment (jbalboni) [inline packages/documentation/src/pages/add-external-pages.mdx]: This is great! Glad we have documentation for how this works already.

**Pull request merged**